react-native-libraries.json

```
@@ -11557,5 +11557,17 @@
     "android": true,
     "web": true,
     "expoGo": true
+  },
+  {
+    "githubUrl": "https://github.com/ro-mgh/mallaui/tree/main/packages/cli",
+    "npmPkg": "mallaui",
+    "examples": [
+      "https://github.com/ro-mgh/mallaui?tab=readme-ov-file#examples",
+      "https://www.mallaui.io/"
+    ],
+    "ios": true,
+    "android": true,
+    "expoGo": true,
+    "newArchitecture": true
```

```suggestion
    "newArchitecture": true,
    "dev": true
```

Let's tag the CLI package with a `dev` flag:
* https://github.com/react-native-community/directory?tab=readme-ov-file#dev

---

Ok) thanks for your participation!